samples/asgardeo-react-app/index.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Vite + React + Asgardeo</title>
```

Comment:
Let's update the title.

```suggestion
    <title>Sample react based SPA with Asgardeo Auth React SDK</title>
```